csharp/ql/src/change-notes/2022-08-10-sqlinjection-queries.md

```diff
@@ -0,0 +1,5 @@
+---
+category: minorAnalysis
+---
+* Added better support for the SQ Lite framework in the SQL injection query.
```

Comment:
Nitpicking: There's an extra space in `SQ Lite`.

Reply:
Thanks for catching it! Will update it!